.github/workflows/ubuntu.yml

```diff
@@ -231,6 +235,11 @@ jobs:
           batch-name: short-tests
           os: ${{ env.os }}
 
+      - name: Run cucumber on short-tests
+        uses: ./.github/workflows/cucumber-tests
+        with:
+          feature: "features/short_tests.feature"
```

Comment:
here partial clone Antares_Simulator_Tests_NR ( only short tests sub dir)

Reply:
or benchmark whether an archive is suitable ( something like [simtest](https://github.com/AntaresSimulatorTeam/SimTest/releases/tag/v9.2.0c) asset but without reference-results

Reply:
I understand your point of view. However, I am reluctant to create an implicit adherence between the feature file naming (or tags) and the structure of the Antares_Simulator_Tests_NR file structure. I think it can possibly lead to confusion, especially since no explicit rule exists to prevent, for example, running a test under "medium-tests" from the "short_test" feature file.